crds/operators.coreos.com_catalogsources.yaml

```diff
@@ -975,11 +975,10 @@ spec:
                         configured to use *must* be using the file-based catalogs in order to utilize this feature.
                       type: object
                       required:
-                        - cacheDir
                         - catalogDir
                       properties:
                         cacheDir:
-                          description: CacheDir is the directory storing the pre-calculated API cache.
+                          description: CacheDir is the (optional) directory storing the pre-calculated API cache.
```

Comment:
Won’t we need to update all the places that use it to ensure the value is set when it isn’t provided?

Reply:
The oddity of the o-f/api repo is that its testing is really in other repos.  Testing for this is in https://github.com/operator-framework/operator-lifecycle-manager/pull/3556